WebApp/static/DIRAC/JobMonitor/classes/JobMonitor.js

```diff
@@ -1135,46 +1129,61 @@ Ext.define('DIRAC.JobMonitor.classes.JobMonitor', {
 
         var oNow = new Date();
 
-        me.plotSettings.plotTitle.text = sTitle + " (" + oNow.toString() + ")"; 
-        me.plotSettings.plotLegend.position = sLegendPosition;
-        
-        var plot = Ext.create("Ext.chart.Chart", {
-              height : 410,
+        me.plotSettings.plotTitle.text = sTitle + " (" + oNow.toString() + ")";
+        me.plotSettings.plotLegend.docked = sLegendPosition;
+        var colors = [];
+        for (var i = 0; i < me.statisticsSelectionGrid.getStore().getCount(); i++) {
+          colors.push(me.statisticsSelectionGrid.getStore().getAt(i).data.color);
+        }
+        me.statisticsSelectionGrid.plot = Ext.create("Ext.chart.PolarChart", {
+              width : 400,
+              height : 400,
               padding : '10 0 0 0',
-              cls : 'jm-statistics-plot-background',
+              theme : 'Base:gradients',
+              layout : 'center',
               animate : true,
               shadow : false,
+              insetPadding : 80,
+              background : {
+                type : 'image',
+                src : GLOBAL.ROOT_URL + 'static/core/img/wallpapers/dirac_jobmonitor_background.png'
```

Comment:
did you change this in #285 ?

Reply:
when #285 will be merged then it will be propagated to integration and I will update this PR

Reply:
ah, I just see this is the background of the pie chart plots. This will stay...